README.md

```diff
@@ -1,158 +1,270 @@
-
-## Troubleshooting ##
-If you experience a problem or bug, please check the Ewon Ignition Demo wiki for more information by visiting [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "https://github.com/hms-networks/FlexyIgnitionDemo/wiki").
-
-If you are unable to find information on your issue, or your issue persists, please contact HMS Networks by emailing [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or create an issue on the Ewon Ignition Demo GitHub project by visiting [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "https://github.com/hms-networks/FlexyIgnitionDemo/issues").
+<!DOCTYPE html>
+<html>
+<body>
+  <!-- HEADER/NAME -->
+  <h1 id="name"><span style="font-variant: small-caps;">Ewon Ignition Demo</span></h1>
+
+  <!-- TABLE OF CONTENTS -->
+  <h2 id="toc"><span style="font-variant: small-caps;">Table of Contents</span></h2>
+
+  <ol>
+    <li><a href="#information">Information</a>
+      <ol>
+        <li><a href="#about">About</a></li>
+        <li><a href="#system-design">System Design</a></li>
+        <li><a href="#trademark-information">Trademark Information</a></li>
+      </ol>
+    </li>
+    <li><a href="#getting-started">Getting Started</a>
+      <ol>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+      </ol>
+    </li>
+    <li><a href="#installation">Installation</a>
+      <ol>
+        <li><a href="#ewon-setup">Ewon Setup</a></li>
+        <li><a href="#ignition-setup">Ignition Setup</a></li>
+      </ol>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#troubleshooting">Troubleshooting</a>
+      <ol>
+        <li><a href="#cannot-log-in-trial-expired">Cannot log in, trial expired</a></li>
+        <li><a href="#error-writing-to-tag-bad_notfound">Error writing to tag, Bad_NotFound<a/></li>
+      </ol>
+    </li>
+    <li><a href="#appendices">Appendices</a>
+      <ol>
+        <li><a href="#setting-up-an-ewon">Setting Up an Ewon</a></li>
+        <li><a href="#obtaining-a-talk2m-developer-id">Obtaining a Talk2M Developer ID</a></li>
+        <li><a href="#accessing-an-ewon-devices-webpage">Accessing an Ewon Device's Webpage</a></li>
+        <li><a href="#installing-and-configuring-ignition">Installing and Configuring Ignition</a></li>
+        <li><a href="#restarting-ignition">Restarting Ignition</a></li>
+        <li><a href="#obtaining-an-ewons-name">Obtaining an Ewon's Name</a></li>
+      </ol>
+    </li>
+  </ol>
+
+  <!-- INFORMATION -->
+  <h2 id="information"><span style="font-variant: small-caps;">Information</span></h2>
+
+  <!-- INFORMATION/ABOUT -->
+  <h3 id="about"><span style="font-variant: small-caps;">About</span></h3>
+  <p>Ewon Ignition Demo is a project that showcases the features of integrating the HMS Networks’ Ewon platform and the Inductive Automation Ignition platform.</p>
```

Comment:
consider dropping "is a project that".  It reads just as well if not better saying "The Ewon Ignition Demo showcases.."

Reply:
Would changing 

integrating the HMS Networks’ Ewon platform and the Inductive Automation Ignition platform

to

integrating HMS Networks’ Ewon platform with Inductive Automation's Ignition SCADA system.

work?

Reply:
Changed